src/components/tag/tag.styles.ts

```diff
@@ -78,7 +78,7 @@ export default css`
 
   .tag--small {
     font-size: var(--sl-button-font-size-medium);
-    height: calc(var(--sl-input-height-small) * 0.9);
+    height: calc(var(--sl-input-height-small) * 0.85715);
```

Comment:
Is this because Inter is slightly different from their default font?

Reply:
This is more because we decided to make our input heights slightly different from the Shoelace default, so if we apply the same * .8 calc to our input heights, we'll get a decimal value height for our tags, which we wanted to avoid.